plugins.go

```diff
@@ -91,6 +91,10 @@ func (p *Plugins) Retrieve() (err error) {
 			continue
 		}
 
+		if len(pi.gitURL) >= 1 && pi.gitURL[0] == '.' {
```

Comment:
I'm torn, although this is perfectly acceptable - I feel like `strings.Index(pi.gitURL, "./") > -1` is technically more correct. I'm going to leave this as a suggestion, rather than a requirement for approval.

Reply:
To ensure that the relative path has to be at the current folder level, and not a parent level? I feel like "../other-repo/plugin" should be valid, no?

Reply:
i think we would want to update other-repo in this case?

Reply:
we support building without `go get` or `git pull` when using `vpm build` instead of `vpm update`

This may be the preferred method for external repo adjustment?